api/v1/authentication_types.go

```diff
@@ -22,26 +22,31 @@ import (
 
 type AuthMethod string
 
-const (
-	AuthMethodOIDC AuthMethod = "OIDC"
-)
-
 // AuthenticationSpec defines the desired state of Authentication
 type AuthenticationSpec struct {
-	// Method configures the method of authentication used by Kibana.
-	// Default: Basic
-	// +kubebuilder:validation:Enum=Basic;OIDC
-	// +required
-	Method AuthMethod `json:"method,omitempty"`
```

Comment:
I can't remember - did we decide it was OK to remove this field entierly? Any existing users which are setting this field will start getting errors where they previously did not.

Reply:
Looking back it seems we would deprecate the field.

It is mentioned in the comments of https://docs.google.com/document/d/1TNaoURVG-6zM2ReBZ2Ky-p1IhbpAjR8a8V_SoZDuhDU/edit# that this should be deprecated, not removed. I will rectify this.

Reply:
I added deprecation. Thanks for bringing this up.

Reply:
@rene-dekker do you mean you added a release note? or something else?